src/pages/iou/request/step/IOURequestStepScan/index.native.js

```diff
@@ -161,7 +161,7 @@ function IOURequestStepScan({
                     return;
                 }
 
-                Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_PARTICIPANTS.getRoute(iouType, transactionID, reportID));
+                Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_CONFIRMATION.getRoute(iouType, transactionID, reportID));
```

Comment:
These changes are still not reverted.

Reply:
@abzokhattab Merging main did not revert these. Please revert these changes so that we can approve the PR.